mesheryctl/internal/cli/root/app/app.go

```diff
@@ -69,20 +69,17 @@ func init() {
 func getSourceTypes() error {
 	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrLoadConfig(err)
```

Comment:
Revert this changes as error returned over here `config.GetMesheryCtl(viper.GetViper())` is already a meshkit error from config package and we need use meshkit logger here